db/re/item_db_equip.yml

```
@@ -83443,7 +83443,19 @@ Body:
     EquipLevelMin: 95
     Refineable: true
     Script: |
-      bonus2 bAddRace,RC_Player_Human,80; bonus2 bIgnoreDefRaceRate,RC_Player_Human,30; .@r = getrefine(); if(.@r>=6) { bonus2 bAddRace,RC_Player_Human,40; } if(.@r>=9) { autobonus2 "{ bonus bShortWeaponDamageReturn,20; bonus bMagicDamageReturn,20; }",6000,2000,BF_WEAPON,"{ specialeffect2 EF_REFLECTSHIELD; }"; }
+      bonus2 bAddRace,RC_Player_Human,80;
+      bonus2 bAddRace,RC_Player_Doram,80;
+      bonus2 bIgnoreDefRaceRate,RC_Player_Human,30;
+      bonus2 bIgnoreDefRaceRate,RC_Player_Doram,30;
+      .@r = getrefine();
+      if(.@r>=6) {
+         bonus2 bAddRace,RC_Player_Human,40;
+         bonus2 bAddRace,RC_Player_Doram,40;
+      }
+      if(.@r>=9) {
+         bonus bShortWeaponDamageReturn,20;
```

```suggestion
         bonus bShortWeaponDamageReturn,20;
         bonus bMagicDamageReturn,20;
```
according to aegis db

---

thanks